apps/docs/components/MDX/billing/billing-details.mdx

```diff
@@ -0,0 +1,10 @@
+Billing details cannot be changed once an invoice is issued, so it's crucial to configure them correctly from the start. You can update your billing details on the  [organization's billing page](https://supabase.com/dashboard/org/_/billing).
+
+### Billing email address
```

Comment:
Not sure if these three things should be in it's own subheading, think they can just be regular sentences maybe?

Reply:
Yeah, you're right. That's not ideal. Will change that and might drop the "Manage your billing details" page completely / move that tiny bit of content somewhere else.

Reply:
I made some changes. Not happy with page "Manage your billing details". Leaving it for now since I don't have a better idea of where to put that info instead.